insecure/net/network_common_test.go

```diff
@@ -1,6 +1,6 @@
-package corruptible
+package net
```

Comment:
It's possible for the test files but unfortunately it's not possible for `insecure/net/network_test_helpers.go` because Go doesn't understand it's a test helper but treats it as a normal Go file and causes compile error if you try and change the package there to `net_test`.

Reply:
This proved too complicated because 2 test helper files (`insecure/net/testhelpers/network_test_helpers.go` and `insecure/net/testhelpers/mock_attacker.go`) can't have a `net_test` package because they are treated like a normal Go file and the compiler won't allow changing their package in the same directory.

Tried to move `insecure/net/testhelpers/network_test_helpers.go` and `insecure/net/testhelpers/mock_attacker.go` to a separate `testhelpers` sub-package under `net` but that caused compilation errors `import cycle not allowed in test` because `insecure/net/testhelpers/network_test_helpers.go` imports the parent package `"github.com/onflow/flow-go/insecure/net"`.